WordPress/src/main/java/org/wordpress/android/ui/JetpackUtils.java

```diff
@@ -0,0 +1,38 @@
+package org.wordpress.android.ui;
+
+import org.wordpress.android.analytics.AnalyticsTracker;
+
+import java.util.HashMap;
+
+/**
+ * Wraps utility methods for Jetpack
+ */
+public class JetpackUtils {
+    private JetpackUtils() {
+    }
+
+    /**
+     * Adds source as a parameter to the tracked Stat
+     * @param stat to be tracked
+     * @param source of tracking
+     */
+    static void trackWithSource(AnalyticsTracker.Stat stat, JetpackConnectionWebViewActivity.Source source) {
```

Comment:
Do you think both `trackWithSource` and `trackFailureWithSource` are reusable elsewhere in the app for other stats, or do you think it's just a Jetpack connection thing?

My take is that, since we have the source parameter, this is just a Jetpack connection thing (that's kind of why I liked the `JetpackConnectionUtils` name in the first place) and if we are going to keep the generic class name, do you think we should name these in a more specific way? Maybe we can consider removing the `stat` parameter and have 2 different success methods (and a private helper method) so the class calling this doesn't have to know about it?

I feel like there are some improvements we can make here, but none of it is too important. I just wanted to share my thoughts on them.

Reply:
I think it's safe for now to have this as a non-reusable method (anyone can change it when the need it). I'll try to groom it a bit.

Reply:
Yeap, I definitely agree that it's safe (and better) to keep it as non-reusable. I brought that up to explain why I think it might be better to use a more specific naming. The class name change did help a lot with this btw and this is a lot less important now.